bootstrapper/inputs/inputs.html

```diff
@@ -22,6 +22,12 @@
 <div>
 	<label><a href="https://github.com/SamGraber/TypeScript-Angular-Components/tree/master/source/components/select/select.md">Select</a>:</label>
 	<rl-select ng-model="input.dropdownSelection" get-options="input.getOptions()" selector="'name'"></rl-select>
+	<rl-select ng-model="input.dropdownSelection" get-options="input.getOptions()" selector="'name'">
+		<div class="row">
+			<div class="col-sm-6">{{$item.name}}</div>
```

Comment:
I made it the default. Comes from `repeat="$item in select.options"` in the rl-select template.